### PR TITLE
addresses CR 1.1 candidate review from Oracle  

### PR DESCRIPTION
**Pull request by sebastiankb**
see #1548

- [ ] RFC2119 assertions
- [ ] hint terms
- [x] vocabulary terms and values MUST be case sensitive

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1558.html" title="Last updated on Aug 17, 2022, 3:47 PM UTC (3e2ba00)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1558/103f786...3e2ba00.html" title="Last updated on Aug 17, 2022, 3:47 PM UTC (3e2ba00)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1168,7 +1168,10 @@ <h2>Class Definitions</h2>
         </p>
 
         <p>
-            Please note that all vocabulary terms and values are case sensitive.
+            <span class="rfc2119-assertion" id="td-processor">
+                Please note that all vocabulary terms and values MUST be case sensitive.
+            </span>
+            
             This is also true for the serialization of the information model (Section <a href="#sec-td-serialization"></a>).
```
Comment (mlagally) [inline index.html:1175]: This looks like another normative requirement. Suggest to either include it in the <span> section above or define another normative <span>

File: index.html [hunk 2]
```diff
@@ -7013,7 +7016,7 @@ <h5>Script Injection</h5>
 	      when generating HTML: with care. 
 	      <span class="rfc2119-assertion" id="sec-inj-sanitize">
 		      Strings sourced from TDs MUST either be sanitized using a carefully vetted HTML
```
Comment (mlagally) [inline index.html:7018]: There was a problem with the markup tooling that prevents having two assertions in the same <span>. Please check with @mmccool  on how to handle that.

File: index.template.html [hunk 3]
```diff
@@ -1168,7 +1168,10 @@ <h2>Class Definitions</h2>
         </p>
 
         <p>
-            Please note that all vocabulary terms and values are case sensitive.
+            <span class="rfc2119-assertion" id="td-processor">
+                Please note that all vocabulary terms and values MUST be case sensitive.
```
Comment (mlagally) [inline index.template.html:1172]: Does this really hold for values?

Are 10m different from 10M ?

File: index.template.html [hunk 4]
```diff
@@ -1168,7 +1168,10 @@ <h2>Class Definitions</h2>
...
+            </span>
+            
             This is also true for the serialization of the information model (Section <a href="#sec-td-serialization"></a>).
         </p>
```
Comment (mlagally) [inline index.template.html:1176]: see comment above

File: index.template.html (hunk 3 above)
Comment (danielpeintner) [reply index.template.html:1172]: @mlagally I think the point you are raising goes beyond what the TD can offer. You are asking for a *real* canonicalization process.
I guess no-one would argue that besides case-sensitive charterers we would also need to match the number "1" with "1.0" etc. Hence, also values need to match "exactly"...

**Comment by egekorkan** [comment]
I will continue working on this PR

**Comment by egekorkan** [comment]
@mlagally , I have addressed your comments that I think are valid and need fixing. There are some that I do not agree with, which I have pointed out in the related issue. Since you have found some normatively important fixes, this PR has normative changes. I think none are very important for a discussion.

File: index.template.html (hunk 3 above)
Comment (sebastiankb) [reply index.template.html:1172]: I think the case sensitive requirement is different to the canonicalization one.  Canonicalization will be a topic for 2.0.

File: index.html (hunk 1 above)
Comment (egekorkan) [reply index.html:1175]: This comment seems to be outdated. This is now a normative requirement. Resolving

**Comment by egekorkan** [comment]
@sebastiankb I am going to resolve the merge conflicts and merge the main here so that we can see the new changes that happened with some assertions

**Comment by egekorkan** [comment]
@sebastiankb I think that this PR is ready. Let's remove WIP label, merge it and then continue work on separate PRs

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge

**Pull request merged**